### ISSUE TITLE
!! SKIOVOX WILL FALL

### ISSUE THREAD

**Issue by Bladebob20000** [bypassiwastaken/skiovox-helper#62]
Not sure when, but skiovox's exploit method has been flagged as an official bug. Chrome devs are working to fix this, so enjoy skiovox while you can. I advise everyone to start using the caub anti autoupdate so that skiovox never gets patched. LOOK AT THE TN DISCORD FOR MORE INFO.

**Comment by 3kh0** [comment]
ok

**Comment by bypassiwastaken** [comment]
yeah i was the first to tell everyone about this. i know. also not the place to talk about it

**Comment by happylabdab2** [comment]
sorry
https://chromium-review.googlesource.com/c/chromium/src/+/5046569
